source/MaterialXGenMdl/Nodes/BlurNodeMdl.cpp

```diff
@@ -61,131 +61,136 @@ void BlurNodeMdl::outputSampleArray(const ShaderGenerator& shadergen, ShaderStag
     shadergen.emitScopeEnd(stage, true);
 }
 
+// No definitions needed for blur in MDL so override the base class with an empty function definition emitter.
+void BlurNodeMdl::emitFunctionDefinition(const ShaderNode&, GenContext&, ShaderStage&) const
+{
+}
```

Comment:
This addition is the only change to this file, all other changes below is just an indentation change so please ignore that.